lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -374,6 +375,51 @@ def test_masquerade_as_specific_student(self):
         self.login_student()
         self.assertEqual(self.get_progress_detail(), u'2/2')
 
+    @patch.dict('django.conf.settings.FEATURES', {
+        'DISABLE_START_DATES': False,
+        'ENABLE_UNICODE_USERNAME': True
```

Comment:
I don't think enabling `ENABLE_UNICODE_USERNAME` would matter here, since it mostly affects the user facing views. Creating a user via a factory should work anyway.

Maybe just keep a comment about it.